DayrSuryan/DSEthiop10.xml

```diff
@@ -380,26 +378,35 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <handNote xml:id="h1" script="Ethiopic">
                         <locus target="#1v"/>
                           <seg type="script"/>
-                          <note>The crude hand of the added beginning of the text
-                            on <ref target="#1v"/>.</note>
+                          <note>Crude later hand of the incipit of <ref target="#ms_i1"/> and
+                          <ref target="#a2"/> on <ref target="#a1"/>.</note>
```

Comment:
do you mean "on" or "and"? The sentence is not clear to me.

Reply:
Ok, modified.